src/Entity/Type/RefundEnumType.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\RefundPlugin\Entity\Type;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Types\ConversionException;
+use Doctrine\DBAL\Types\Type;
+use Sylius\RefundPlugin\Model\RefundType;
+
+final class RefundEnumType extends Type
+{
+    public function getName(): string
+    {
+        return 'refund_type';
```

Comment:
What about naming it `sylius_refund_refund_type`? `sylius_refund` being the vendor and `refund_type` the name. It will protect us from conflicts with any other custom DBAL type.